app/src/main/java/com/example/plantsapp/presentation/ui/tasks/TasksFragment.kt

```diff
@@ -80,9 +82,18 @@ class TasksFragment : Fragment(R.layout.fragment_tasks) {
                 layoutManager = LinearLayoutManager(context)
                 adapter = plantsWithTasksAdapter
             }
+
+            layoutNoTasks.ivNoItemsImage.setImageResource(R.drawable.ic_baseline_calendar_24)
+            layoutNoTasks.tvNoItemsTitle.setText(R.string.title_no_tasks)
+            layoutNoTasks.tvNoItemsDetail.setText(R.string.msg_no_tasks)
```

Comment:
In such cases I write Extension function on ***Binding , to have one method that will change all needed data